CHANGES

```diff
@@ -4,6 +4,11 @@
 
  Changes between 1.0.2g and 1.1.0  [xx XXX xxxx]
 
+  *) --strict-warnings no longer enables runtime debugging options
+     such as REF_DEBUG. Instead, REF_DEBUG is enabled automatically
```

Comment:
Indeed, there's more to follow up with here. Thanks for the review; levitte@ wanted to take a look as well so I'll wait, in case he has good ideas about cleaning up --debug. Right now what you get seems to depend on the compiler. (The various confs in 10-main.conf have different debug pickers.)